src/tools/fuzzing/random.cpp

```diff
@@ -20,8 +20,8 @@
 
 namespace wasm {
 
-Random::Random(std::vector<char>&& bytes, FeatureSet features)
-  : bytes(std::move(bytes)), features(features) {
```

Comment:
I'm really surprised if this was incorrect. This is part of the initializer list rather than the constructor body, so I would expect it to properly initialize the member `bytes` in terms of the parameter `bytes`. This works for `features(features)`, so why wouldn't it work for `bytes(std::move(bytes))`?

Reply:
Oh no, this is fine per se, but then `bytes` on line 27 ends up pointing to the parameter `bytes`, not the member variable `bytes`. The other way we can fix this is to leave this name as is and change line 27 to `this->bytes`. Let me know if you prefer that.

Reply:
ohhh, makes sense.